packages/slate-editor/src/modules/nodes-hierarchy/queries/isTextualNode.ts

```diff
@@ -0,0 +1,8 @@
+import { isHeadingNode, isParagraphNode } from '@prezly/slate-types';
+import type { Node } from 'slate';
+
+import { isInlineNode } from './isInlineNode';
+
+export function isTextualNode(node: Node) {
+    return isInlineNode(node) || isParagraphNode(node) || isHeadingNode(node);
```

Comment:
> `isInlineNode(node)`

Do we have to unwrap all inline nodes? What about placeholders? Right now it's possible to use them in the caption:

![Screenshot from 2022-08-24 12-20-23](https://user-images.githubusercontent.com/370680/186381479-d5e16537-7244-4e86-9682-f8f2055fe659.png)

Reply:
Well, it is an inline node, maybe there can be special cases that users will want to have placeholders. I am not sure that such cases will occur, but do not see a reason why should we forbid it.

Reply:
>  but do not see a reason why should we forbid it

But this change here will unwrap the pasted placeholders, as they are inline. Right?

Reply:
unwrapping does not working for placeholder because it has not child elements